rust-toolchain.toml

```diff
@@ -1,5 +1,5 @@
 [toolchain]
-channel = "1.70.0"
+channel = "stable"
```

Comment:
Do we also need to change `RUST_TOOLCHAIN` in `release.yml`? (It is still set to 1.70.0)

Reply:
Also in follow up pr.